Tasks/JenkinsQueueJobV2/task.json

```diff
@@ -110,6 +110,24 @@
             "required": false,
             "helpMarkDown": "Specifies strictness of a success definition: whether to consider unstable as a failure or not. False for non-strict, and true for strict version.",
             "groupName": "advanced"
+        },
+        {
+            "name": "retryNumber",
```

Comment:
Could you please align naming with already existing inputs for other tasks - like Copy Files? ([docs](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/CopyFilesV2/task.json#L94))

Reply:
Sure, sounds good.